src/main/java/goodgenerator/crossmod/thaumcraft/Research.java

```diff
@@ -184,7 +185,37 @@ public static void addResearch() {
                                 Arrays.asList(
                                         new TC_Aspects.TC_AspectStack(TC_Aspects.AQUA, 256),
                                         new TC_Aspects.TC_AspectStack(TC_Aspects.VACUOS, 256),
-                                        new TC_Aspects.TC_AspectStack(TC_Aspects.PERMUTATIO, 128))) });
+                                        new TC_Aspects.TC_AspectStack(TC_Aspects.PERMUTATIO, 128))),
+                        GregTech_API.sThaumcraftCompat.addInfusionRecipe(
+                                "ESSENTIA_CELL",
+                                ItemRefer.Essentia_Cell_T4.get(1),
+                                new ItemStack[] {
+                                        GT_OreDictUnificator.get(OrePrefixes.plate, Materials.MysteriousCrystal, 1),
+                                        ItemList.Emitter_UV.get(1L),
+                                        GT_ModHandler.getModItem(Mods.Avaritia.ID, "Resource", 1, 5),
+                                        ItemList.Circuit_Board_Wetware.get(1),
+                                        GT_ModHandler.getModItem(Mods.TaintedMagic.ID, "ItemMaterial", 1, 5),
+                                        GT_ModHandler.getModItem(Mods.ThaumicTinkerer.ID, "kamiResource", 1, 1) },
+                                ItemRefer.Essentia_Cell_T5.get(1),
+                                10,
+                                Arrays.asList(
+                                        new TC_Aspects.TC_AspectStack(TC_Aspects.AQUA, 512),
+                                        new TC_Aspects.TC_AspectStack(TC_Aspects.VACUOS, 512),
+                                        new TC_Aspects.TC_AspectStack(TC_Aspects.PERMUTATIO, 256))),
+                        GregTech_API.sThaumcraftCompat.addInfusionRecipe(
+                                "ESSENTIA_CELL",
+                                ItemRefer.Essentia_Cell_T5.get(1),
```

Comment:
Going to put this out there, Glod himself expressed disdain with the LEG going through so many tiers, and if this also has laser support, it will make the balancing ridiculous, even with jank